march-roundup.md

```diff
@@ -0,0 +1,26 @@
+| Author | TallAmericano |
+| :---- | :---- |
+| Title | March Roundup |
+| Headline  | Looking at the news in March. |
+| Cypherpunktimes | link |
+| Publish Date | xx Mar 2024 |
+
+| Social Media | Link |
+| :---- | :---- |
+| Twitter | twitter link |
+| Twitter | thread link… |
+| Twitter | poll link… |
+| Reddit  | link… |
+| Bison Relay | News feed title |
+| Other platforms | Instagram and TikTok |
+
+Thesis statement - This article looks at the news in March and interesting topics
+
+Pre-discussion, research and questions
+
```

Comment:
Coinbase outage during BTC ATH (Tuesday 5th March 2024) could be an interesting piece to cover for this month's news. Why does Coinbase seem to struggle when the price is exploding, could there be an underlying liquidity issue?

Reply:
Maybe if we did a few articles about coinbase and the vulnerabilities it could be good. They might list us in exchange for the negative press haha.

Reply:
Context is everything. I don’t think it benefits us to post articles that promote fud, even if it does promote the values of non custodial exchanges. 

That being said, this is where writing full descriptions in this news pieces comes into play. A great opportunity to tell the whole story but as news rather than fud.